test/browser/run-test.sh

```diff
@@ -4,25 +4,66 @@ set -eux
 # tests need cockpit's bots/ libraries and test infrastructure
 cd $SOURCE
 git init
+rm -f bots  # common local case: existing bots symlink
 make bots test/common
 
-# only install a subset to save time/space
-rm -f package-lock.json  # otherwise the command below installs *everything*, argh
-npm install chrome-remote-interface sizzle
+# support running from clean git tree
+if [ ! -d node_modules/chrome-remote-interface ]; then
+    npm install chrome-remote-interface sizzle
+fi
+
+# disable detection of affected tests; testing takes too long as there is no parallelization,
+# and TF machines are slow and brittle
+mv .git dot-git
 
 . /etc/os-release
 export TEST_OS="${ID}-${VERSION_ID/./-}"
+export TEST_AUDIT_NO_SELINUX=1
 # HACK: upstream does not yet know about rawhide
-if [ "$TEST_OS" = "fedora-34" ]; then
-    export TEST_OS=fedora-33
+if [ "$TEST_OS" = "fedora-35" ]; then
+    export TEST_OS=fedora-34
 fi
 
-export TEST_AUDIT_NO_SELINUX=1
-
 EXCLUDES=""
 
+# triggers SELinux violation
+# See journal: SELinux is preventing /usr/libexec/qemu-kvm from open access on the file /var/lib/cockpittest/nfs_pool/nfs-volume-0.
+EXCLUDES="$EXCLUDES TestMachinesDisks.testAddDiskNFS"
+
+# Fedora gating tests are running on infra without /dev/kvm; Machines tests are too darn slow there
+# so just pick a representative sample
+if [ "$ID" = "fedora" ]; then
+    # Testing Farm machines are really slow in European evenings
+    export TEST_TIMEOUT_FACTOR=3
+    TESTS="TestMachinesConsoles.testInlineConsole
+           TestMachinesCreate.testCreateImportDisk
+           TestMachinesCreate.testCreatePXE
+           TestMachinesDisks.testAddDiskDirPool
+           TestMachinesDisks.testAddDiskNFS
+           TestMachinesDisks.testAddDiskSCSI
+           TestMachinesLifecycle.testBasic
+           TestMachinesNICs.testNICAdd
+           TestMachinesNetworks.testNetworks
+           TestMachinesSettings.testBootOrder
+           TestMachinesSettings.testVCPU
+           TestMachinesSnapshots.testSnapshots
```

Comment:
You don't run anything from the storage test here. But this list existed already, rigth?

Reply:
I actually did in my earlier attempts, but these are still [too broken](http://artifacts.dev.testing-farm.io/dc53c16c-7c61-4f46-8ada-d395937d2e55/) in TF. I'd really like to add them at some point, but do that step by step.

Note that in cockpit we only run a *single* create test in packit -- so that list is already a big improvement.

Reply:
FTR: This is not meant to be "The Final List" -- I do want to build it up gradually. But I think it's a good enough start to be useful.